mappings/services.json

```diff
@@ -1,4 +1,27 @@
 {
+  "settings": {
+    "index": {
+      "number_of_shards": 1
+    },
+    "analysis": {
+      "analyzer": {
+        "stemming_analyzer": {
+          "tokenizer": "standard",
+          "filter": ["standard", "lowercase", "possessive_english_stemmer", "light_english_stemmer"]
+        }
+      },
+      "filter": {
+        "light_english_stemmer": {
+          "type": "stemmer",
+          "name": "light_english"
+        },
+        "possessive_english_stemmer": {
```

Comment:
Wrong link, try this: https://trello.com/c/Y6rdUgxU/517-search-stemming#comment-59785039d8758610c70b78d1